azure-pipelines-release.yml

```
@@ -150,3 +150,40 @@ stages:
         git commit -am "Bump Home Assistant $version"
         git push
       displayName: 'Update version files'
+  - job: 'ReleaseDocker'
+    pool:
+      vmImage: 'ubuntu-latest'
+    steps:
+    - script: |
+        set -e
+        export DOCKER_CLI_EXPERIMENTAL=enabled
+
+        sudo docker manifest create homeassistant/home-assistant:$(Build.SourceBranchName) \
+          homeassistant/amd64-homeassistant:$(Build.SourceBranchName) \
+          homeassistant/i386-homeassistant:$(Build.SourceBranchName) \
+          homeassistant/armhf-homeassistant:$(Build.SourceBranchName) \
+          homeassistant/armv7-homeassistant:$(Build.SourceBranchName) \
+          homeassistant/aarch64-homeassistant:$(Build.SourceBranchName)
+
+        sudo docker manifest annotate homeassistant/home-assistant:$(Build.SourceBranchName) \
+          homeassistant/amd64-homeassistant:$(Build.SourceBranchName) \
+          --os linux --arch amd64
+
+        sudo docker manifest annotate homeassistant/home-assistant:$(Build.SourceBranchName) \
+          homeassistant/i386-homeassistant:$(Build.SourceBranchName) \
+          --os linux --arch i386
+
+        sudo docker manifest annotate homeassistant/home-assistant:$(Build.SourceBranchName) \
+          homeassistant/armhf-homeassistant:$(Build.SourceBranchName) \
+          --os linux --arch arm32v6
+
+        sudo docker manifest annotate homeassistant/home-assistant:$(Build.SourceBranchName) \
+          homeassistant/armv7-homeassistant:$(Build.SourceBranchName) \
+          --os linux --arch arm32v7
```

Isn't it better to set variants?

```suggestion
          --os linux --arch arm --variant arm32v7
```